<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TitleView</class>
 <widget class="QWidget" name="TitleView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1046</width>
    <height>41</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TitleView</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>1</number>
   </property>
   <property name="leftMargin">
    <number>5</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>5</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <widget class="QToolButton" name="menu_btn">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="ToolButton2State" name="play_module_btn">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="ToolButton2State" name="netpage_module_btn">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>866</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QToolButton" name="min_btn">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="max_btn">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="restore_btn">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="close_btn">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ToolButton2State</class>
   <extends>QToolButton</extends>
   <header>toolbutton2state.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
